megamock/megamocks.py

```diff
@@ -155,26 +171,27 @@ class _MegaMockMixin(Generic[T]):
 
     def __init__(
         self,
-        spec: T | None = None,
+        spec: T | type[T] | None = None,
         *,
+        wraps: T | None = None,
+        spy: T | None = None,
+        spec_set: bool = True,
+        instance: bool | None = None,
+        side_effect: Any = None,
+        return_value: Any = MISSING,
```

Comment:
since return_value is being passed, it is always part of the kwargs, so MISSING is needed to differentiate